receiver/awscloudwatchreceiver/config.go

```diff
@@ -62,6 +63,7 @@ var (
 	errInvalidPollInterval            = errors.New("poll interval is incorrect, it must be a duration greater than one second")
 	errInvalidAutodiscoverLimit       = errors.New("the limit of autodiscovery of log groups is improperly configured, value must be greater than 0")
 	errAutodiscoverAndNamedConfigured = errors.New("both autodiscover and named configs are configured, Only one or the other is permitted")
+	errPrefixAndPatternConfigured     = errors.New("both prefix and pattern configs are configured, Only one or the other is permitted")
```

Comment:
I see this is consistent with the above error, but I think it's very unreadable, and would recommend it be changed to match the readme.

```suggestion
	errPrefixAndPatternConfigured     = errors.New("cannot specify both prefix and pattern")
```